src/storage/src/hummock/sstable_store.rs

```diff
@@ -190,8 +260,18 @@ impl SstableStore {
             };
             let data_path = self.get_sst_data_path(sst.id);
             let store = self.store.clone();
+            let sst_id = sst.id;
 
             async move {
+                if let Some(holder) = tiered_cache
```

Comment:
Do we need to check `CachePolicy` and skip getting from `tiered_cache` if the policy is `CachePolicy::Disable`?

Reply:
I'll fix that.